ui/desktop/app/services/store.js

```diff
@@ -3,7 +3,10 @@
  * SPDX-License-Identifier: BUSL-1.1
  */
 
-import Store, { CacheHandler } from '@ember-data/store';
+// This an example of using Store from ember-data/store to get default configuration automatically.
+// eslint-disable-next-line ember/use-ember-data-rfc-395-imports
+import Store from 'ember-data/store';
```

Comment:
Comparing with the `addons/api/addon/services/store` file, I see this one is importing store from `ember-data/store` and the other is `@ember-data/store`.

My understanding is that when importing from @ember-data we need to set defaults per ourselves?

I also left a comment/question within the `api/addon` store file.